harvard-lts/CURIOSity
# Using spotlight-oaipmh-resources 3.0.2.

**Using spotlight-oaipmh-resources 3.0.2.**
* * *

# What does this Pull Request do?
This updates the Gemfile to use spotlight-oaipmh-resources 3.0.2.

# How should this be tested?

A description of what steps someone could take to:
* Rebuild the container using the `gem-version-3-0-2` branch
* sh into the container: `docker exec -it curiosity-web-1 sh`
* Confirm version 3.0.2 of the gem is installed: `bundle list` should show: `spotlight-oaipmh-resources (3.0.2 ad83603)`
* Harvest the ming exhibit to confirm it is still working properly.

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? No
- integration tests? No
